src/components/controls/counter/Counter.css

```diff
@@ -1,26 +1,9 @@
-.clock-setter {
+.clock-starter {
```

Comment:
Try to use `rem` for padding here.

Reply:
Thanks I will definitively change everywhere